rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/common/event/dto/RMStateDTO.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * ProActive Parallel Suite(TM):
+ * The Open Source library for parallel and distributed
+ * Workflows & Scheduling, Orchestration, Cloud Automation
+ * and Big Data Analysis on Enterprise Grids & Clouds.
+ *
+ * Copyright (c) 2007 - 2017 ActiveEon
+ * Contact: contact@activeeon.com
+ *
+ * This library is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation: version 3 of
+ * the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * If needed, contact us to obtain a release under GPL Version 2 or 3
+ * or a different license than the AGPL.
+ */
+package org.ow2.proactive.resourcemanager.common.event.dto;
+
+import java.io.Serializable;
+import java.util.List;
+
+import org.ow2.proactive.resourcemanager.common.event.RMNodeEvent;
+import org.ow2.proactive.resourcemanager.common.event.RMNodeSourceEvent;
+
+
+public class RMStateDTO implements Serializable {
+
+    private long latestCounter;
+
+    private List<RMNodeSourceEvent> nodeSource;
+
+    private List<RMNodeEvent> nodesEvents;
+
+    public long getLatestCounter() {
+        return latestCounter;
+    }
+
+    public void setLatestCounter(long latestCounter) {
+        this.latestCounter = latestCounter;
+    }
+
+    public List<RMNodeSourceEvent> getNodeSource() {
```

Comment:
Can we rename to `getNodeSourceEvents` or to `getNodeSourcesEvents`?

Reply:
No, here we cannot rename without changing clients